docs/photoCredits.txt

```diff
@@ -0,0 +1,17 @@
+PHOTO CREDITS:

```

Comment:
The photoCredits.txt file should really be named PHOTO-CREDITS (or PHOTO-CREDITS.md if you want to use markdown formatting).  This is in keeping with the convention of other meta files in a project, like LICENSE, AUTHORS, README, etc.